src/Microsoft.AspNetCore.OData/Routing/ODataActionSelector.cs

```diff
@@ -107,15 +116,16 @@ public ActionDescriptor SelectBestCandidate(RouteContext context, IReadOnlyList<
                         .Where(p =>
                         {
                             return p.ParameterType != typeof(ODataPath) &&
-                                !ODataQueryParameterBindingAttribute.ODataQueryParameterBinding.IsODataQueryOptions(p.ParameterType);
```

Comment:
>                                 [](start = 0, length = 32)

did you change the spacing intentionally?  All things being equal, I prefer it indented.